lib/routes.py

```diff
@@ -10,7 +9,7 @@
 # Test group room route
 @app.route('/groups', methods=['POST'])
 def add_group():
-  body = request.form
+  body = request.json
   if body['name'] != '' and body['description'] != '':
```

Comment:
Have we settled that FE is sending us JSON for sure?  They work very similar just wanted to confirm.